data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/PeerForwarderConfiguration.java

```diff
@@ -64,6 +65,7 @@ public PeerForwarderConfiguration() {}
     public PeerForwarderConfiguration (
             @JsonProperty("port") final Integer serverPort,
             @JsonProperty("request_timeout") final Integer requestTimeout,
+            @JsonProperty("client_timeout") final Integer clientTimeout,
```

Comment:
Should the configuration name also be `client_timeout_millis`? Other option is to reuse the `DataPrepperDurationDeserializer` class

Reply:
I wanted to remain consistent with `request_timeout`, which also in millis. Ideally all of these durations would use the DataPrepper duration but that may be a breaking change